### ISSUE TITLE
font size

### ISSUE THREAD

**Issue by tune007** [johannesjo/super-productivity-android#42]
### :speaking_head: 

please add an option to change the font size of the Android app

**Comment by jiongxuan** [comment]
@johannesjo Is it possible to set the font size in the WebApp so that it can be supported on all platforms?

I also feel that the text size on Android is a bit small (English and other languages)

**Comment by johannesjo** [comment]
Currently there is only the option to make everything smaller or bigger using the browsers zoom level. Setting a different font size is something we could add on web level as well, but it might be a lot of work to make all the components work with other than the default, so I'd prefer sticking to the general zoom level approach.

**Comment by jiongxuan** [comment]
@johannesjo OK, got it. We can first understand everyone's needs and consider a common scaling level solution.

**Comment by johannesjo** [comment]
> @johannesjo OK, got it. We can first understand everyone's needs and consider a common scaling level solution.

Yes, I think it will be easier to maintain. Not sure if you knew, but it's also already implemented for the electron via ctrl + mousewheel. :)
